src/mbgl/tile/raster_dem_tile.cpp

```diff
@@ -20,31 +20,6 @@ RasterDEMTile::RasterDEMTile(const OverscaledTileID& id_,
       mailbox(std::make_shared<Mailbox>(*Scheduler::GetCurrent())),
       worker(parameters.workerScheduler,
              ActorRef<RasterDEMTile>(*this, mailbox)) {
```

Comment:
For tiles on the bottom edge or top edge, consider updating neighboring tiles to mark those neighbors as backfilled.